### ISSUE TITLE
Shoes packager downloads "shoes-3.3.6-osx-10.9.tgz" when selecting OSX as architecture.

### ISSUE THREAD

**Issue by bolandross** [shoes/shoes3#405]
Hello!

I was just giving Shoes 3.3.6 a try on macOS 10.13.4 High Sierra.

When using the packager and choosing OSX as target architecture, the file `shoes-3.3.6-osx-10.9.tgz` is downloaded.

Unfortunately, this Shoes version doesn't run on newer OSX systems. Can this be changed somehow, so the packager downloads the latest OSX runtime as default?

**Comment by bolandross** [comment]
Thanks for the super-quick answer!

I would love to help, just tell me what to do 😃

To make it easier to troubleshoot, some more info from my side:

On the shoesrb.com website I chose `Shoes 3.3.6 for OSX 10.10 and above`  
then I downloaded and installed it by dragging it into my Applications-folder.
Then I started it and chose _Package an App with Shoes_, and set up everything as shown on the screenshot:
![bildschirmfoto 2018-04-09 um 17 26 35](https://user-images.githubusercontent.com/5481702/38507221-412bca9a-3c1c-11e8-8bd9-d737ff458e00.png)
It seems like the wrong package is chosen automatically.

**Comment by ccoupe** [comment]
Yup, I may not know that 10.10 is higher than 10.9 - quick solution - I've hidden the 10.9 download so give it a try again.

**Comment by bolandross** [comment]
I can confirm that it now works correctly!
